<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>10</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTreeWidget" name="treeWidget">
           <column>
            <property name="text">
             <string>Assem</string>
            </property>
           </column>
          </widget>
          <widget class="QTabWidget" name="tabWidget01">
           <property name="currentIndex">
            <number>2</number>
           </property>
           <widget class="QWidget" name="tabWidgetPage1">
            <attribute name="title">
             <string/>
            </attribute>
           </widget>
           <widget class="QWidget" name="tabWidgetPage2">
            <attribute name="title">
             <string/>
            </attribute>
           </widget>
           <widget class="QWidget" name="tabWidgetPage3">
            <attribute name="title">
             <string>页</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tabWidgetPage4">
            <attribute name="title">
             <string>页</string>
            </attribute>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="occWidget" native="true">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>486</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>导入</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="actionRbtImport"/>
     <addaction name="separator"/>
     <addaction name="actionWopImport"/>
     <addaction name="separator"/>
     <addaction name="actionRbtJointImport"/>
     <addaction name="separator"/>
     <addaction name="actionToolImport"/>
     <addaction name="separator"/>
     <addaction name="actionSTLImport"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExport">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionRbtImport">
   <property name="text">
    <string>导入机器人</string>
   </property>
  </action>
  <action name="actionWopImport">
   <property name="text">
    <string>导入工件</string>
   </property>
  </action>
  <action name="actionRbtJointImport">
   <property name="text">
    <string>导入机器人关节</string>
   </property>
  </action>
  <action name="actionToolImport">
   <property name="text">
    <string>导入工具</string>
   </property>
  </action>
  <action name="actionSTLImport">
   <property name="text">
    <string>导入点云</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
